kubernetes-client-openapi/build.gradle

```diff
@@ -0,0 +1,64 @@
+plugins {
+    id 'io.micronaut.build.internal.kubernetes-module'
+    id 'io.micronaut.openapi' version libs.versions.micronaut.gradle.plugin
+}
+
+micronautBuild {
+    binaryCompatibility {
+        enabled.set(false)
+    }
+}
+
+micronaut {
+    openapi {
+        client(layout.buildDirectory.file("openapi.yaml").get().asFile) {
+            apiPackageName = "io.micronaut.kubernetes.client.openapi.api"
+            modelPackageName = "io.micronaut.kubernetes.client.openapi.model"
+            clientId = "kubernetes-client"
+            useReactive = false
+        }
+    }
+}
+
+dependencies {
+    annotationProcessor(mnValidation.micronaut.validation.processor)
+    annotationProcessor(mnSerde.micronaut.serde.processor)
+    annotationProcessor(mn.micronaut.inject.java)
+    implementation(mnValidation.micronaut.validation)
+    implementation(mnSerde.micronaut.serde.jackson)
+    implementation(mnOpenapi.micronaut.openapi)
+    implementation(libs.bcpkix.jdk18on)
```

Comment:
does this need to be a direct dependency?

Reply:
Removed bouncycastle dependency